### PR TITLE
Try to fix #989

### LINKED ISSUE #989: Warning in torch.lr_scheduler
## 🐛 Bug Report
On catalyst training with scheduler a warning shows.

```
OS: Windows
Python 3.7.8
catalyst==20.10.1
torch==1.6.0
```

### How To Reproduce
```
import torch
from torch import nn
import torch.nn.functional as F
from torch.utils.data import TensorDataset, DataLoader
from catalyst import dl

X = torch.randn(1000, 10)
y = torch.rand(X.shape[0])
model = nn.Linear(X.shape[1], 1)
optimizer = torch.optim.Adam(model.parameters())
scheduler = torch.optim.lr_scheduler.ExponentialLR(optimizer, gamma=0.9)

dataset = TensorDataset(X, y)

loaders = {
    'train': DataLoader(dataset, batch_size=32, shuffle=True),
    'valid': DataLoader(dataset, batch_size=32)
}

class CustomRunner(dl.SupervisedRunner):
    def _handle_batch(self, batch):
        # model train/valid step
        y_pred = self.model(batch['features']).view(-1)

        loss = F.mse_loss(y_pred, batch['targets'])
        self.batch_metrics.update(
            {"loss": loss}
        )

runner = CustomRunner()

runner.train(
    model=model,
    loaders=loaders,
    optimizer=optimizer,
    scheduler=scheduler,
    num_epochs=20
)
```

#### Screenshots
image

### Expected behavior
No warnings

PS
- [x] I know, that I could [join Catalyst slack (#__questions channel)](https://join.slack.com/t/catalyst-team-core/shared_invite/zt-d9miirnn-z86oKDzFMKlMG4fgFdZafw) for issue discussion.

### PR DESCRIPTION
**Pull request by CrafterKolyan**
## Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contribution guide](https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md)?
- [ ] Did you check the code style? `catalyst-make-codestyle && catalyst-check-codestyle` (`pip install -U catalyst-codestyle`).
- [ ] Did you make sure to update the docs? We use Google format for all the methods and classes.
- [ ] Did you check the docs with `make check-docs`?
- [ ] Did you write any new necessary tests?
- [ ] Did you check that your code passes the unit tests `pytest .` ? 
- [ ] Did you add your new functionality to the docs?
- [x] Did you update the [CHANGELOG](https://github.com/catalyst-team/catalyst/blob/master/CHANGELOG.md)?

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

PS
- [x] I know, that I could [join slack](https://join.slack.com/t/catalyst-team-core/shared_invite/zt-d9miirnn-z86oKDzFMKlMG4fgFdZafw) for pull request discussion.

### CODE REVIEW

**Comment by Scitator** [comment]
dear @CrafterKolyan, could you please try to update the minimal torch requirement to 1.3?

File: requirements/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Framework must-haves
 numpy>=1.16.4
-torch>=1.1.0
+torch>=1.5.0
```
Comment (Scitator) [inline requirements/requirements.txt]: could you please try 1.3? It's our maximum minimal requirement during team discussions :)

**Review by Scitator — DISMISSED** [review]

File: requirements/requirements.txt (hunk 1 above)
Comment (CrafterKolyan) [reply requirements/requirements.txt]: @Scitator I've tried and it didn't work. I'll make another fix

File: catalyst/callbacks/scheduler.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 from typing import Optional, Tuple, TYPE_CHECKING
 from abc import ABC, abstractmethod
+from packaging import version
```
Comment (CrafterKolyan) [inline catalyst/callbacks/scheduler.py:4]:
```suggestion

from packaging import version
```

File: catalyst/callbacks/scheduler.py [hunk 3]
```diff
@@ -1,6 +1,9 @@
 from typing import Optional, Tuple, TYPE_CHECKING
 from abc import ABC, abstractmethod
 
+
```
Comment (CrafterKolyan) [inline catalyst/callbacks/scheduler.py]:
```suggestion
```

File: catalyst/callbacks/scheduler.py [hunk 4]
```diff
@@ -118,7 +120,10 @@ def _scheduler_step(
             lr = scheduler.optimizer.param_groups[0]["lr"]
         else:
             scheduler.step()
-            lr = scheduler.get_lr()[0]
+            if version.parse(torch.__version__) < version.parse("1.6.0"):
```
Comment (Scitator) [inline catalyst/callbacks/scheduler.py:123]: could you please add this flag to the setting, so we don't need to recompute it at each call?
our settings file https://github.com/catalyst-team/catalyst/blob/master/catalyst/settings.py
something like `IS_TORCH_ABOVE_16v` would be good enough

**Review by Scitator — CHANGES_REQUESTED** [review]

**Comment by Scitator** [comment]
btw, should be fixed already
